docker-compose.yml

```diff
@@ -130,3 +132,25 @@ services:
     networks:
       - frontend
       - backend
+
+  osmosis:
+    build:
+      context: ./smoketests
+      dockerfile: build.Dockerfile
+    volumes:
+      - ${DATA_PATH_HOST}/pbf:/etc/pbf
+
+  pgtap:
```

Comment:
Where is `pgtap` coming from?

Reply:
pgTAP is the technology behind the unit tests. But this is probably better renamed to `test` or `smoketest`